src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -0,0 +1,297 @@
+        {
+        }
+    }
+
+    public void Dispose() {}
+}
+";
+            string expectedOperationTree = @"
+IUsingStatement (OperationKind.UsingStatement)
+  IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+    IVariableDeclaration: Program p1 (OperationKind.VariableDeclaration)
+      Initializer: IObjectCreationExpression (Constructor: Program..ctor()) (OperationKind.ObjectCreationExpression, Type: Program)
+    IVariableDeclaration: Program p2 (OperationKind.VariableDeclaration)
+      Initializer: IObjectCreationExpression (Constructor: Program..ctor()) (OperationKind.ObjectCreationExpression, Type: Program)
+  IBlockStatement (0 statements) (OperationKind.BlockStatement)
+";
+            VerifyOperationTreeForTest<VariableDeclarationSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void ForLoopDeclaration()
+        {
+            string source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        for (/*<bind>*/int i = 0/*</bind>*/; i < 0; i++)
+        {
+
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IForLoopStatement (LoopKind.For) (OperationKind.LoopStatement)
+  Condition: IBinaryOperatorExpression (BinaryOperationKind.IntegerLessThan) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+      Left: ILocalReferenceExpression: i (OperationKind.LocalReferenceExpression, Type: System.Int32)
+      Right: ILiteralExpression (Text: 0) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 0)
+  Local_1: System.Int32 i
+  Before: IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+      IVariableDeclaration: System.Int32 i (OperationKind.VariableDeclaration)
+        Initializer: ILiteralExpression (Text: 0) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 0)
+  AtLoopBottom: IExpressionStatement (OperationKind.ExpressionStatement)
+      IIncrementExpression (UnaryOperandKind.IntegerPostfixIncrement) (BinaryOperationKind.IntegerAdd) (OperationKind.IncrementExpression, Type: System.Int32)
+        Left: ILocalReferenceExpression: i (OperationKind.LocalReferenceExpression, Type: System.Int32)
+        Right: ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1)
+  IBlockStatement (0 statements) (OperationKind.BlockStatement)
+";
+            VerifyOperationTreeForTest<VariableDeclarationSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void ForLoopMultipleDeclarations()
+        {
+            string source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        for (/*<bind>*/int i = 0, j = 0/*</bind>*/; i < 0; i++)
```

Comment:
>= 0 [](start = 29, length = 3)

Consider also testing with the value missing and with equals value missing (two different scenarios).